addons/l10n_anz_ubl_pint/models/account_edi_xml_pint_anz.py

```diff
@@ -33,7 +34,7 @@ def _get_tax_category_code(self, customer, supplier, tax):
         # OVERRIDE account_edi_ubl_cii
         # A business not registered for GST cannot issue tax invoices.
         # In this case, the tax category code should be O (Outside scope of tax).
-        if not supplier.vat:
+        if not self.env.company.l10n_au_is_gst_registered:
```

Comment:
Maybe it wasn't clear, but you cannot use self.env ^^

I think this should work for now:

```suggestion
        if not supplier.ref_company_ids[:1].l10n_au_is_gst_registered:
```
until in 19.x we support self billing and move the field to the partner.